<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClamUI</class>
 <widget class="QMainWindow" name="ClamUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>500</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <widget class="TabScanScheduling" name="tab_01">
       <attribute name="icon">
        <iconset theme="chronometer">
         <normaloff/>
        </iconset>
       </attribute>
       <attribute name="title">
        <string>Automatische Virenprüfung</string>
       </attribute>
      </widget>
      <widget class="TabVirusScan" name="tab_02">
       <attribute name="icon">
        <iconset theme="system-search">
         <normaloff/>
        </iconset>
       </attribute>
       <attribute name="title">
        <string>Datei(en) auf Viren prüfen</string>
       </attribute>
      </widget>
      <widget class="TabVirusQuarantine" name="tab_03">
       <attribute name="icon">
        <iconset theme="document-encrypted">
         <normaloff/>
        </iconset>
       </attribute>
       <attribute name="title">
        <string>Dateien in Quarantäne</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3"/>
      </widget>
      <widget class="TabVirusDB" name="tab_04">
       <attribute name="icon">
        <iconset theme="document-properties">
         <normaloff/>
        </iconset>
       </attribute>
       <attribute name="title">
        <string>ClamAV Virendefinitionen</string>
       </attribute>
      </widget>
      <widget class="TabAboutClamAV" name="tab_05">
       <attribute name="icon">
        <iconset theme="clamav">
         <normaloff/>
        </iconset>
       </attribute>
       <attribute name="title">
        <string>Über ClamAV</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_Help">
        <property name="text">
         <string>Hilfe</string>
        </property>
        <property name="icon">
         <iconset theme="documentation">
          <normaloff/>
         </iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_About">
        <property name="text">
         <string>Über...</string>
        </property>
        <property name="icon">
         <iconset theme="help-about">
          <normaloff/>
         </iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_Settings">
        <property name="text">
         <string>Einstellungen</string>
        </property>
        <property name="icon">
         <iconset theme="configure">
          <normaloff/>
         </iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_Close">
        <property name="toolTip">
         <string>Schließt dieses Fenster. Das Programm läuft im Systemabschnitt der Kontrollleiste weiter.</string>
        </property>
        <property name="text">
         <string>Schließen</string>
        </property>
        <property name="icon">
         <iconset theme="dialog-close">
          <normaloff/>
         </iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;Datei</string>
    </property>
    <addaction name="action_Close"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;?</string>
    </property>
    <addaction name="action_Handbook"/>
    <addaction name="separator"/>
    <addaction name="actionClamAV_Online"/>
    <addaction name="separator"/>
    <addaction name="action_AboutClamUI"/>
    <addaction name="action_AboutKDE"/>
   </widget>
   <widget class="QMenu" name="menu_Extra">
    <property name="title">
     <string>E&amp;xtra</string>
    </property>
    <addaction name="action_Settings"/>
   </widget>
   <widget class="QMenu" name="menu_ClamAV">
    <property name="title">
     <string>&amp;ClamAV</string>
    </property>
    <widget class="QMenu" name="menuClamAV_Daemon">
     <property name="title">
      <string>ClamAV Dämon</string>
     </property>
     <property name="icon">
      <iconset theme="clamav"/>
     </property>
     <property name="toolTipsVisible">
      <bool>true</bool>
     </property>
     <addaction name="actionClamAV_Daemon_Start"/>
     <addaction name="actionClamAV_Daemon_Stop"/>
    </widget>
    <widget class="QMenu" name="menuFreshClam">
     <property name="title">
      <string>FreshClam</string>
     </property>
     <property name="icon">
      <iconset theme="clamav"/>
     </property>
     <addaction name="actionFreshclam_Daemon_Start"/>
     <addaction name="actionFreshClam_Daemon_Stop"/>
    </widget>
    <addaction name="actionDateien_auf_Viren_pr_fen"/>
    <addaction name="separator"/>
    <addaction name="menuClamAV_Daemon"/>
    <addaction name="menuFreshClam"/>
    <addaction name="separator"/>
    <addaction name="action_Update_VirusDB"/>
    <addaction name="action_FilesQuarantyne"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_ClamAV"/>
   <addaction name="menu_Extra"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action_Close">
   <property name="icon">
    <iconset theme="dialog-close">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Schließen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Beenden</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Handbook">
   <property name="icon">
    <iconset theme="documentation">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Handbuch</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="action_AboutClamUI">
   <property name="icon">
    <iconset theme="help-about">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Über ClamUI</string>
   </property>
  </action>
  <action name="action_AboutKDE">
   <property name="icon">
    <iconset theme="kde">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Über &amp;KDE</string>
   </property>
  </action>
  <action name="action_Settings">
   <property name="icon">
    <iconset theme="configure"/>
   </property>
   <property name="text">
    <string>Einstellungen</string>
   </property>
  </action>
  <action name="actionClamAV_Online">
   <property name="icon">
    <iconset resource="../clamui.qrc">
     <normaloff>:/icons/icons/breeze/actions/toolbar/web-browser.svg</normaloff>:/icons/icons/breeze/actions/toolbar/web-browser.svg</iconset>
   </property>
   <property name="text">
    <string>ClamAV im Web</string>
   </property>
  </action>
  <action name="actionClamAV_Daemon_Start">
   <property name="icon">
    <iconset theme="clamav"/>
   </property>
   <property name="text">
    <string>ClamAV Dämon starten</string>
   </property>
  </action>
  <action name="actionClamAV_Daemon_Stop">
   <property name="icon">
    <iconset theme="clamav"/>
   </property>
   <property name="text">
    <string>ClamAV Dämon anhalten</string>
   </property>
  </action>
  <action name="actionFreshclam_Daemon_Start">
   <property name="icon">
    <iconset theme="clamav"/>
   </property>
   <property name="text">
    <string>FreshClam als Dämon starten</string>
   </property>
  </action>
  <action name="actionFreshClam_Daemon_Stop">
   <property name="icon">
    <iconset theme="clamav"/>
   </property>
   <property name="text">
    <string>FreshClam Dämon anhalten</string>
   </property>
  </action>
  <action name="action_Update_VirusDB">
   <property name="icon">
    <iconset theme="document-properties"/>
   </property>
   <property name="text">
    <string>Virendefinitionen aktualisieren</string>
   </property>
  </action>
  <action name="action_FilesQuarantyne">
   <property name="icon">
    <iconset theme="document-encrypted"/>
   </property>
   <property name="text">
    <string>Dateien in Quarantäne</string>
   </property>
  </action>
  <action name="actionDateien_auf_Viren_pr_fen">
   <property name="icon">
    <iconset theme="system-search"/>
   </property>
   <property name="text">
    <string>Dateien auf Viren prüfen</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TabVirusScan</class>
   <extends>QWidget</extends>
   <header>src/tabvirusscan.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabScanScheduling</class>
   <extends>QWidget</extends>
   <header>src/tabscanscheduling.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabVirusQuarantine</class>
   <extends>QWidget</extends>
   <header>src/tabvirusquarantine.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabVirusDB</class>
   <extends>QWidget</extends>
   <header>src/tabvirusdb.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabAboutClamAV</class>
   <extends>QWidget</extends>
   <header>src/tababoutclamav.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>pushButton_Help</tabstop>
  <tabstop>pushButton_About</tabstop>
  <tabstop>pushButton_Settings</tabstop>
  <tabstop>pushButton_Close</tabstop>
 </tabstops>
 <resources>
  <include location="../clamui.qrc"/>
 </resources>
 <connections/>
</ui>
